examples/mobile/ScheduleApp/css/style.css

```diff
@@ -34,14 +34,51 @@
 }
 
 .content-footer {
-	color: #909090;
+	color: var(--text-secondary-color);
 }
 
 .date-text {
 	font-size: 12px;
 	margin: auto 12px;
+	color: var(--text-secondary-color);
```

Comment:
text color of unselected items  in WebAppReference20313.pdf for dark theme is
#3E91FF, in your case is #999999
but I will clarify this with UX team, let's stay as it is now